### ISSUE TITLE
Should homebrew package specfiy auto_updates?

### ISSUE THREAD

**Issue by RafalSkolasinski** [alienator88/Pearcleaner#122]
**Is your feature request related to a problem? Please describe.**

Apps that auto-updates themselves seems to inform homebrew about that.
For example for `Tailscale` cask you will see
```bash
$ brew info tailscale --cask
==> tailscale: 1.76.0 (auto_updates)
...
```

This prevents `homebrew` from updating these apps.

Not sure if big issue but probably there should be only one way to update Pearcleaner.

**Describe the solution you'd like**

Mark pearcleaner in homebrew as app that has `auto_updates`.

**Describe alternatives you've considered**

**Additional context**

**Comment by alienator88** [comment]
Some users like to update only via homebrew or only via Pearcleaner. The built-in updater can be disabled in settings by setting to Never. 
I'd like to leave the option up to the user I think in this case.
Does it cause any issues without auto_updates set in the cask? I'm not super familiar with that property.

**Comment by RafalSkolasinski** [comment]
Basically I just had a situation that:
- I opened the app, it notified me about the update, I updated
- In terminal I was updating all homebrew stuff, it updated Pearcleaner again.

So I am wondering if that will not cause any issues - the update happening twice. Mainly wanted to check that and as I didn't find any other issues mentioning `auto_updates` I opened this one.

**Comment by alienator88** [comment]
I see. Yeah I mean it shouldn't cause any issues if it updated twice. 
Was the brew update running in terminal at the same time as the app updated itself using the built-in installer?

**Comment by RafalSkolasinski** [comment]
No, no, not at the same time. But depending what the update process involves having an application managed by two "package managers" may cause issues. But if the update only involves a simple bump of a binary then there probably should be no issue. I am new to all the mac and homebrew stuff, just so far every app I've seen that auto-updates itself had that `auto_updates` flag set.

**Comment by alienator88** [comment]
I'll have to look into the `auto_updates` flag, I'm not very familiar with it. If it makes sense for my setup I'll definitely consider adding it :) 
Thank you!

**Comment by alienator88** [comment]
I looked more into this and I think it does make sense to add this flag to the cask code.
I didn't really think of the scenario where if Pearcleaner was updated using the built-in installer, homebrew is not aware of that external update and still thinks it's on the older version causing a duplicate update to happen during brew upgrade command.
Next time I push an update out to homebrew, I'll get that flag added on.
Thanks!
